.gitignore

```diff
@@ -15,4 +15,5 @@
 # vendor/
 
 # golang
-.idea/
\ No newline at end of file
+.idea/
+main
```

Comment:
Why we ignore main here?

Reply:
The executable main file created as a result of running `go build -o main cmd/mqtt-blackbox-exporter/main.go`.
I was testing the changes, and forgot to remove this line.

Reply:
No problem, let's keep it there from you. Again, thanks for your idea 🚀

Reply:
You're welcome. :heart: